src/rich-text-editor.component.js

```diff
@@ -162,10 +165,31 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
   }
 })
 
+// This hook allows you to change the focused value synchronously instead of queuing it
```

Comment:
this is lifechanging feedback I know but `queueing`

Reply:
oh haha good catch